scripts/build_experimental

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "${SCRIPTS_DIR}/common"
+
+cargo build "${cargo_build_flags[@]}"  --manifest-path=experimental/Cargo.toml
```

Comment:
`cargo clippy` and `cargo test` too somewhere?

Reply:
Done.